client/src/pages/HomePage.jsx

```diff
@@ -1,10 +1,127 @@
+import { useEffect, useState } from 'react';
+
+import { env } from '../../config/env';
+
+import axios from 'axios';
+
 import Navbar from '../components/Navbar';
 
+const userID = '6737bd5921b1fac154eadf76';
+
+import './mycss.css';
+
+import { findBestDisplayDecks } from '../utils/deckProgress';
+
+import { addTimesCompleted } from '../utils/addDeckProgress.js';
+
 export default function HomePage() {
+  const [dailyWords, setDailyWords] = useState([]);
+  const [dailyMotivation, setDailyMotivation] = useState('');
+  const [user, setUser] = useState('');
+  const [displayDecks, setDisplayDecks] = useState([]);
+
+  // You want data? This will give you data
+  useEffect(() => {
+    axios.get(`${env.API_URL}/users/${userID}`).then((res) => {
```

Comment:
Maybe move this to the services folder. It can be re-used by others. You can also use the api helper so you don't have to rely on env.js:
```
import api from ../services/index.js

api.get(/users/${userID}`).then...

```